### ISSUE TITLE
Not able to list polices on management group

### ISSUE THREAD

**Issue by murarisumit** [Azure/azure-cli#9067]
**Describe the bug**
Not able to list polices on management group.

```bash
az policy assignment list \
--disable-scope-strict-match \
--scope "/providers/Microsoft.Management/managementGroups/my-mg-group" 
--query [].name
```

**To Reproduce**

Steps to reproduce the behavior.

1. Create a policy
```bash
 az policy definition create \
   --management-group my-mg-group
   --name 'sumit-enforce-nsg-on-subnett' \
   --display-name 'Sumit- NSG X on every subnet' \
   --description 'This policy enforces a specific NSG on every subnet' \
   --rules <azurepolicy.rules.json> \
   --params <azurepolicy.parameters.json> \
   --mode All 
```

2. Assign the policy to management group.
```bash
az policy assignment create \
  --name "Force nsg in subnet" \
  --scope "/providers/Microsoft.Management/managementGroups/my-mg-group" \
  --policy "/providers/Microsoft.Management/managementgroups/my-mg-group/providers/Microsoft.Authorization/policyDefinitions/sumit-enforce-nsg-on-subnett" \
  -p '{"nsgId": { "value":"sumit-test-vm-nsg"}}'
```

below command should work, but not working for me.

```bash
az policy assignment list \
--disable-scope-strict-match \
--scope "/providers/Microsoft.Management/managementGroups/my-mg-group" 
--query [].name
```

**Expected behavior**
I should be able to list the assigned polices in management group

**Environment summary**

Install : pip
CLI version: azure-cli                         2.0.60 *
OS version: macOS Mojave 10.14.4
Shell type: bash

**Additional context**

**Comment by achandmsft** [comment]
Hi Sumit (@murarisumit) ,
Thanks a lot for your bug report! Could you please send us what error you are getting when you try to list the assigned policies? 

Thanks,
Arun Chandrasekhar
Principal Program Manager
Azure Management Experience

**Comment by murarisumit** [comment]
Sorry for late reply, but when I just went into scope without `--disable-scope-strict-match` flag. I am getting empty list. 

When trying with `--disable-scope-strict-match` flag, I am getting policies from different scope, but still not the policy that I'm able to view in azure UI portal and created by using `az policy assignment create` command.

**Comment by yonzhan** [comment]
add to S164.

**Comment by zhoxing-ms** [comment]
We are already upgrading this issue, you can work around it for a while in this way:
`az rest --method get --uri "https://management.azure.com/providers/Microsoft.Management/managementGroups/{managementGroups}/providers/Microsoft.Authorization/policyAssignments?api-version=2019-09-01&$filter=atScope()"`

**Comment by zhoxing-ms** [comment]
@pilor  Please pay attention to this issue. After upgrading the service and SDK, please let us know. Thanks

**Comment by pilor** [comment]
ack

**Comment by yonzhan** [comment]
@pilor any update for this issue?

**Comment by pilor** [comment]
There is a problem with the azure-mgmt-resource python package generation in recent versions which is preventing me from generating a package with the changes I need.  @zikalino is looking into it.

**Comment by yonzhan** [comment]
@pilor  nice to know, please let us know when the PR is ready.  @zhoxing-ms  to follow up.

**Comment by pilor** [comment]
@zhoxing-ms issue is fixed in the linked PR

**Comment by yonzhan** [comment]
@zhoxing-ms please check the PR to fix the issue.

**Comment by zhoxing-ms** [comment]
@murarisumit Hello, this issue has been fixed in the latest version of azure-cli (2.2.0), please download the latest version of the package to try.
If there are any questions, please feel free to give us feedback~
